security/src/main/scala/pl/touk/nussknacker/ui/security/oidc/OidcAuthenticationConfiguration.scala

```diff
@@ -90,6 +95,7 @@ object OidcAuthenticationConfiguration {
   import net.ceedubs.ficus.readers.EnumerationReader._
   import pl.touk.nussknacker.engine.util.config.CustomFicusInstances._
   import pl.touk.nussknacker.ui.security.api.AuthenticationConfiguration.authenticationConfigPath
+  private implicit val valueReader: ValueReader[FrontendStrategySettings] = forDecoder
```

Comment:
WDYT about moving the implicit val to the `create` method?

Reply:
And what about implicit imports? should be in the `create` as well? Looks like it is the only usage of them.

Reply:
IMO it'd be nice to move them too. E.g. when we remove the `create` method in the future, it'd be easier to keep the code clean.